example/App.tsx

```diff
@@ -69,10 +114,10 @@ class BannerTest implements Test {
 
   render(onMount: (component: any) => void): React.ReactNode {
     return (
-      <View style={styles.testSpacing} ref={onMount}>
+      <View ref={onMount}>
         <BannerAd
           unitId={bannerAdUnitId}
-          size={BannerAdSize.FULL_BANNER}
+          size={BannerAdSize.ADAPTIVE_BANNER}
```

Comment:
cool - I saw the suggestion and liked it - I'm still not personally using mobile ads though I will soon, until then I trust  your judgement completely